chicago/templates/about.html

```diff
@@ -10,15 +10,15 @@
         <br />
         <h1 id='about'>About</h1>
 
-        <p><strong>Chicago Councilmatic</strong> tracks all things related to Chicago City Council: the <a href='/search/' target='_blank'>legislation</a> introduced and passed, its various <a href='/committees/' target='_blank'>committees</a> and the <a href='/events/' target='_blank'>meetings</a> they hold, and the <a href='/council-members/' target='_blank'>aldermen</a> themselves.</p>
```

Comment:
we used these `target="_blank"` tags everywhere. we shouldn't use these for internal links